github-api-call-inventory/README.md

```diff
@@ -0,0 +1,37 @@
+# GitHub API Call Rate Limits
+
+## Inventory of GitHub API calls made during a pipeline run
+
+See [this Jupyter Notebook](./github-api-call-inventory.ipynb).
+
+## Collecting metrics on GitHub API calls
+
+We want to count the number of GitHub API calls, so that we can produce an
+indicator about the average API calls per pipeline run. Such an indicator
+could help us notice if code changes result in an increase in the API calls
+made, and so making it more likely that the call rate limits are reached in
+certain orgs.
```

Comment:
I'd also appreciate seeing the status codes of those requests so we know how many of them are `ok`, `4xx`, `2xx`